tests/meta/test_dists.py

```diff
@@ -1,14 +1,13 @@
 import tarfile
 import zipfile
 from pathlib import Path
-from sys import version_info
+from subprocess import CompletedProcess, run
+from sys import executable, version_info
```

Comment:
```
import subprocess
import sys
```

I prefer just importing the module. And doesn't seem like much more work to do `sys.executable` for example.

Google's style guide also recommends this, with a few exceptions:
https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
I actually had that before for subprocess, but black mangles the formatting. I guess at least for typing purposes I'd like to have concise imports to not explode functions. But just skimming through that chapter, I'm not sure you'd like some of the rest of google's style recommendations on imports :grin: